### DISCUSSION TITLE
Undocumented theme property: `ui.statusline.active`

### DISCUSSION THREAD

**Discussion by tryoxiss** [helix-editor/helix#10466] [General]
The theme property `"ui.statusline.active"` does not appear anywhere on the [themeing documentation](https://docs.helix-editor.com/themes.html), I found out it exists by guessing based off the inactive property (I was going to suggest it if it didnt exist). 

Every case of `ui.statusline`
![image](https://github.com/helix-editor/helix/assets/94419893/738e66ef-6101-433a-97da-5b378905215a)

but adding a `.a` leads to zero results
![image](https://github.com/helix-editor/helix/assets/94419893/5bafc6e0-402b-4669-a9a9-254b8d9e8077)

**Comment by Netetra** [comment]
Is anyone working on this project right now? I'd like to make my first contribution.

**Comment by tryoxiss** [comment]
I think since nobody has responded yet its safe to say nobody is working on it !
